theme/styles/snippets/product-grid-item.scss

```diff
@@ -36,4 +36,14 @@
       max-height: 17rem;
     }
   }
+
+  &__vendor-avatar {
+    height: 1.5rem;
```

Comment:
These values seem to be used a few times, might make sense to make them in to scss vars (same with lg-up)